ua-converter/requirements.txt

```diff
@@ -1,3 +1,5 @@
+-r ../requirements.txt
```

Comment:
This definitely should not be in the requirements.txt file `-r ../requirements.txt`

Reply:
We need this to import `fake_useragent`.